deploy/roles/r/defaults/main.yml

```diff
@@ -22,3 +24,9 @@ r_env_vars:
   CC: /opt/rh/devtoolset-2/root/usr/bin/gcc
   CPP: /opt/rh/devtoolset-2/root/usr/bin/cpp
   CXX: /opt/rh/devtoolset-2/root/usr/bin/c++
+
+r_packages_with_opts:
+  -
+    name: udunits2
```

Comment:
Minor quibble, but the `name:` key can go on the same line as the `-`, which improves readability a bit.